config.json

```diff
@@ -1,820 +1,818 @@
 {
-  "language": "Rust",
-  "blurb": "Rust is a compiled programming language designed for speed, concurrency, and memory safety. Rust programs can run almost anywhere, from low-power embedded devices to web servers.",
   "active": true,
+  "blurb": "Rust is a compiled programming language designed for speed, concurrency, and memory safety. Rust programs can run almost anywhere, from low-power embedded devices to web servers.",
   "exercises": [
     {
-      "uuid": "13ec1ebe-d71b-436f-ab12-25305e814171",
-      "slug": "hello-world",
       "core": false,
-      "unlocked_by": null,
       "difficulty": 1,
+      "slug": "hello-world",
```

Comment:
slug seems important. it seems like it should come first or second (behind uuid)